README.md

```diff
@@ -49,6 +49,14 @@ QuestDB release and sample datasets:
 - live trading data from a cryptocurrency exchange
 - geolocations of 250k unique ships over time
 
+| Query                                                                         | Execution time                                                                                                                                                                   |
+| ----------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `SELECT sum(double) FROM 1.6bn`                                               | [0.15 secs](<https://demo.questdb.io/?query=SELECT%20sum(trip_distance)%20FROM%20trips;&executeQuery=true>)                                                                      |
```

Comment:
Not a big fan of using FROM 1.6bn instead of the real table name. We might want to add a new column with the title "scanned rows" and put 1.6bn, 115M..., but the 1.6bn in the query looks misleading, specially as it wouldn't be valid SQL

Reply:
Good point. I guess it's fine if I use `trips` here then, as we already specified we have 1.6bn records above this table.